scene/gui/tabs.cpp

```diff
@@ -340,21 +347,36 @@ void Tabs::_notification(int p_what) {
 					max_drawn_tab = i;
 				}
 
-				Rect2 sb_rect = Rect2(w, 0, tabs[i].size_cache, h);
+				Rect2 sb_rect = Rect2(w, y, tabs[i].size_cache, font->get_height() + font->get_ascent());
 				sb->draw(ci, sb_rect);
 
 				w += sb->get_margin(MARGIN_LEFT);
 
 				Size2i sb_ms = sb->get_minimum_size();
 				Ref<Texture> icon = tabs[i].icon;
+
 				if (icon.is_valid()) {
 
-					icon->draw(ci, Point2i(w, sb->get_margin(MARGIN_TOP) + ((sb_rect.size.y - sb_ms.y) - icon->get_height()) / 2));
+					if (position == BOTTOM) {
+						icon_y = get_size().height - icon->get_height() / 2;
+					}
+					else {
```

Comment:
`else {` should be on the same line as the previous `}` as per our style guide, see https://travis-ci.org/godotengine/godot/jobs/417610905 and http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html